java/google/registry/whois/WhoisHttpServer.java

```diff
@@ -144,7 +145,9 @@ public void run() {
       String command = decode(JOINER.join(SLASHER.split(path.substring(PATH.length())))) + "\r\n";
       Reader reader = new StringReader(command);
       DateTime now = clock.nowUtc();
-      sendResponse(SC_OK, new WhoisReader(reader, now).readCommand().executeQuery(now));
+      sendResponse(SC_OK, new WhoisReader(reader, commandFactory, now)
```

Comment:
Line wrapping here is a little wonky - does this not just all fit on one line?  If not, prefer to format as

    sendResponse(
        SC_OK,
        new WhoisReader(reader, commandFactory, now).readCommand().executeQuery(now));

So the syntactic structure is more evident.